lib/mi_bridges/driver.rb

```diff
@@ -142,7 +142,6 @@ def debug(e)
       # rubocop:disable Debugger
       binding.pry if ENV["DEBUG_DRIVE"]
       # rubocop:enable Debugger
-      raise e
```

Comment:
by not re-raising we can continue to step through the app even if a debugger statement arises. Is that problematic?

Reply:
I would suggest, instead, something like

```ruby
if ENV["DEBUG_DRIVE"]
  binding.pry
else
  raise e
end
```

That way we can continue only if we're in debug mode. Which I think is fair.

Reply:
done!